include/stumpless/entry.h

```diff
@@ -26,6 +26,7 @@
 #  include <stdarg.h>
 #  include <stdbool.h>
 #  include <stddef.h>
+#  include <stdlib.h>
```

Comment:
This header addition does not seem relevant to your change, you should be able to remove it.